### PR TITLE
Feature/unload document

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: assets/dev/js/editor/editor-base.js [hunk 1]
```diff
@@ -786,13 +786,13 @@ export default class EditorBase extends Marionette.Application {
 
 		this.addDeprecatedConfigProperties();
 
+		elementorCommon.elements.$window.trigger( 'elementor:loaded' );
```
Comment (kobizz) [inline assets/dev/js/editor/editor-base.js]: Consider swaping of 'elementor:loaded' and 'elementor:init' events

Comment (matipojo) [reply assets/dev/js/editor/editor-base.js]: Added as a task.

File: core/common/assets/js/api/core/commands.js [hunk 2]
```diff
@@ -72,6 +72,38 @@ export default class Commands extends elementorModules.Module {
 		return this;
 	}
 
+	unregister( component, command ) {
```
Comment (matipojo) [inline core/common/assets/js/api/core/commands.js]: Test

File: core/common/assets/js/api/core/shortcuts.js [hunk 3]
```diff
@@ -51,6 +51,16 @@ export default class Shortcuts {
 		} );
 	}
 
+	unregister( shortcuts, args ) {
```
Comment (matipojo) [inline core/common/assets/js/api/core/shortcuts.js]: Test

File: core/common/assets/js/api/modules/component-base.js [hunk 4]
```diff
@@ -108,6 +108,10 @@ export default class ComponentBase extends elementorModules.Module {
 		$e.routes.register( this, route, callback );
 	}
 
+	unregisterRoute( route ) {
```
Comment (matipojo) [inline core/common/assets/js/api/modules/component-base.js]: Test

**Pull request merged**